markdown.dtx

```diff
@@ -23396,7 +23405,7 @@ function M.reader.new(writer, options)
   parsers.Str      = (parsers.normalchar * (parsers.normalchar + parsers.at)^0)
                    / writer.string
 
-  parsers.Symbol   = (V("SpecialChar") - parsers.tightblocksep)
+  parsers.Symbol   = (parsers.backtick^1 + V("SpecialChar") - parsers.tightblocksep)
```

Comment:
@lostenderman To add `parsers.backtick` to `V("SpecialChar")`, please use `reader->add_special_character(parsers.backtick)`: https://github.com/Witiko/markdown/blob/45ee7f11aa8167c0e901d71a77e64165d6db7ca0/markdown.dtx#L24301-L24306.